projects/js-packages/charts/src/components/line-chart/stories/index.docs.mdx

```diff
@@ -275,7 +274,7 @@ Charts are fully keyboard accessible with Tab navigation and arrow key data poin
 
 Display series information with automatic color matching:
 
-<Canvas of={ LegendStories.Default } />
+<Canvas of={ LineChartStories.CustomLegendPositioning } />
 
 <Source
 	language="jsx"
```

Comment:
Canvas and source don't match here

![Screenshot 2025-08-06 at 2 47 11 PM](https://github.com/user-attachments/assets/2c88fba4-716a-4fdd-b4c5-4bb62a5e3829)